cli/options/options.go

```diff
@@ -73,6 +73,13 @@ var ConfigFile = cli.StringFlag{
 	Usage: "path to the node configuration file (overrides --config-path option)",
 }
 
+// RelativePath is a flag for commands that use node configuration and provide
+// a prefix to all relative paths in config files.
+var RelativePath = cli.StringFlag{
+	Name:  "relative-path",
+	Usage: "a prefix to all relative paths in config files",
```

Comment:
s/`in config files`/`in the node configuration file`, just to match the description of other config-related options.